src/renderer/components/ft-subscribe-button/ft-subscribe-button.js

```diff
@@ -105,6 +105,10 @@ export default defineComponent({
       const currentProfile = deepCopy(profile)
 
       if (this.isProfileSubscribed(profile)) {
+        // confirmation pop-up
+        if (!confirm('Are you sure you want to unsubscribe?')) {
```

Comment:
Could you update `static/locales/en-US` to add this text to it and use the `this.$t` function? This would allow this text to be translated.

Ex: https://github.com/FreeTubeApp/FreeTube/blob/833355bbbd4f0cb57139f20ef02a45729c0f9c8b/src/renderer/components/ft-subscribe-button/ft-subscribe-button.js#L117

Reply:
Done

Reply:
I think you may have forgotten to push the changes to `static/locales/en-US.yaml`

Reply:
Its already there in en-US.yaml in line no. 127
![Screenshot from 2024-04-08 14-29-42](https://github.com/FreeTubeApp/FreeTube/assets/148802868/702b8e86-4db8-48d3-8f18-6e6c8a33a4fc)